test/common/network/utility_fuzz_test.cc

```diff
@@ -0,0 +1,89 @@
+#include "envoy/config/core/v3/address.pb.h"
+
+#include "common/network/address_impl.h"
+#include "common/network/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  {
+    const std::string string_buffer(reinterpret_cast<const char*>(buf), len);
```

Comment:
It looks like this same string is constructed multiple times. Can we do it once and share the instance to avoid the extra memory copy operations?

Reply:
Yes, fixed in latest commit.